src/_includes/banner.html

```diff
@@ -8,9 +8,5 @@
     See how Flutter is pushing UI development at Flutter Forward;<br>
     streaming live from Kenya on January 25, 2023.
     <a href="https://flutter.dev/events/flutter-forward">Register now</a>
-    {% comment %}
-    <br>The Flutter and Dart teams are hiring.
-    <a href="https://docs.flutter.dev/jobs">Learn more</a><br>
-    {% endcomment %}
```

Comment:
This is already commented out, so it doesn't even show up in the HTML as a comment. I feel no need to delete it.